resources/aws_ec2.go

```diff
@@ -146,6 +156,33 @@ type chanStruct struct {
 	err   error
 }
 
+// snsPolicy denotes the structure of sns security policies.
+type snsPolicy struct {
+	Version   string      `json:"Version"`
+	ID        string      `json:"Id"`
+	Statement []statement `json:"Statement"`
+}
+
+// statement denotes the structure of sns security policy statements.
+type statement struct {
```

Comment:
does it make sense to call this snsStatement to make it more obvious what it's related to?